### PR TITLE
Yet another upstream merge

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Carbonhell** [comment]
no,-1,your upstream merges fucking break everything
did you even test this

**Comment by ghost** [comment]
testing right now

File: _maps/map_files/generic/HippieCentcomm.dmm [hunk 1]
```diff
@@ -8123,10 +8123,10 @@
 /turf/closed/wall/mineral/plastitanium,
 /area/shuttle/syndicate/airlock)
 "vt" = (
-/obj/machinery/porta_turret/syndicate{
-	dir = 5
+/obj/effect/turf_decal/stripes/line{
```
Comment (GuyonBroadway) [inline _maps/map_files/generic/HippieCentcomm.dmm:8126]: You see this this shit? This shit right here? You make sure shuttles work for this shit. I'm going to leave this here til,l you confirm all the following works.

1: calling a shuttle and riding it to centcom.
2: Changing a shuttle and riding it to centcom.
3: Riding a shuttle as antag so you end up going to the syndie base.
4: doing so with a different shuttle.

**Review by GuyonBroadway — CHANGES_REQUESTED** [review]

File: _maps/map_files/generic/HippieCentcomm.dmm (hunk 1 above)
Comment (ghost) [reply _maps/map_files/generic/HippieCentcomm.dmm:8126]: this was just copy/pasting the new non-lopsided syndieshuttle...

Comment (GuyonBroadway) [reply _maps/map_files/generic/HippieCentcomm.dmm:8126]: You're still going to make sure all this works.

Comment (ghost) [reply _maps/map_files/generic/HippieCentcomm.dmm:8126]: I am

**Review by ghost — CHANGES_REQUESTED** [review]
Do not merge untill another maintainer has tested this, and confirmed it works

**Comment by GuyonBroadway** [comment]
YOU SEE THIS SHIT?
fixthisshit
FIX IT!!

**Comment by GuyonBroadway** [comment]
Holy fuck Amari Ray quit us so this upstream is ded in the fukken water. Closing unless he wants to try it later and open it.

**Comment by ghost** [comment]
this codebase is dead in the fukken water, mate

**Comment by GuyonBroadway** [comment]
I'll take that as a no.

**Pull request closed without merging**